modules/gke/variables.tf

```diff
@@ -22,13 +22,13 @@ variable "require_squad" {
 }
 
 variable "squad" {
-  description = "squad label to apply to the service."
+  description = "Squad label to apply to the service."
```

Comment:
Note: Should we replace the squad with team?

Reply:
I'm happy to, in this PR, if that's the way we're going?

Reply:
i think that will require more things, thanks!